doc/whats_new/v0.24.rst

```diff
@@ -783,7 +783,7 @@ Changelog
 - |Enhancement| :class:`model_selection.GridSearchCV`,
   :class:`model_selection.RandomizedSearchCV` and
   :func:`model_selection.cross_validate` support `scoring` being a callable
-  returning a dictionary of of multiple metric names/values association.
+  returning a dictionary of multiple metric names/values association.
```

Comment:
Same here regarding [ci failing](https://app.circleci.com/jobs/github/scikit-learn/scikit-learn/190617?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). Can you revert this change?